### DISCUSSION TITLE
2.11.5

### DISCUSSION THREAD

**Discussion by akallabeth** [FreeRDP/FreeRDP#9782] [Q&A]
Noteworthy changes:
* Fix integer overflow in progressive decoder
* Update OpenSSL API usage for compatiblility with newer versions (#9747)
* Prevent NULL dereference for single thread decoder (#9712)

For a complete and detailed change log since the last release run:
git log 2.11.5...2.11.4

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeRDP/FreeRDP/releases/tag/2.11.5'>2.11.5</a>.</em>

**Comment by MatteoBax** [comment]
If I configure the project on Windows by running this command:
```
cmake -S freerdp-2.11.5 -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded -B freerdp-build  -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON -DWITH_SERVER=ON -DWITH_SAMPLE=ON -DWITH_PLATFORM_SERVER=OFF -DWITH_FFMPEG=ON -DWITH_SWSCALE=ON -DWITH_DSP_FFMPEG=ON -DWITH_WEBVIEW=OFF -DCMAKE_INSTALL_PREFIX=D:\freerdp -DBUILD_SHARED_LIBS=OFF -DCMAKE_PREFIX_PATH=D:\Dev_Libs
```
I get 
```
CMake Error at CMakeLists.txt:865 (message):
  FFMPEG support requested but not detected


-- Configuring incomplete, errors occurred!
```
If I add
```
include(FindFFmpeg)
```
 to the CMakeList.txt file, the problem is solved

**Reply by akallabeth** [reply]
@MatteoBax juse ensure that:

1. your `CMake` is recent enough to check `pkg-config` and similar tools in `CMAKE_PREFIX_PATH` 
2. set your `CMAKE_PREFIX_PATH` to include the `FFMpeg` (and other dependency) libraries, include paths, `CMake` or `PkgConfig` file paths, ...

no code changes required.
